lib/main.js

```diff
@@ -205,6 +205,7 @@ module.exports = class Router extends EventEmitter {
                     type,
                     sourceMaps,
                     minify,
+                    env: process.env.NODE_ENV,
```

Comment:
it would be great if this would be an option to the constructor

Reply:
'tis but the work of a moment

Reply:
I shall dip it in yoghurt and cover it with chocolate buttons

Reply:
🍫